src/core/ext/filters/client_channel/lb_policy/xds/xds_cluster_manager.cc

```diff
@@ -466,7 +462,9 @@ void XdsClusterManagerLb::ClusterChild::Orphan() {
   // the child.
   picker_wrapper_.reset();
   if (delayed_removal_timer_handle_.has_value()) {
-    engine_->Cancel(*delayed_removal_timer_handle_);
+    xds_cluster_manager_policy_->channel_args()
```

Comment:
This LB policy should use the same pattern as the grpclb and weighted_target LB policies: The LB policy should be given a `std::shared_ptr<EventEngine>` data member that will be set in its ctor using `GetObjectRef<>()`, and the nested classes can access that data member directly instead of fishing it out of channel args themselves.

Reply:
Good call, done. I've also renamed the Subchannel member to keep things consistent.